src/app/view/_routes/editorganization/editorganization.component.ts

```diff
@@ -123,7 +143,7 @@ export class EditOrganizationComponent implements OnInit {
             .addPassword(this.formArray.value[1].orgPwdCtrl)
             .build(),
         );
-      for (let i = 0; i < this.mapDataChild.arrayCoord.length; i++) {
+      for (let i = 0; i < this.mapDataChild.arrayCoord?.length; i++) {
```

Comment:
dato che non usate mai "i" direttamente, questo starebbe meglio come foreach invece che for

Reply:
qui usiamo i per iterare su tutti gli array avendo la stessa lunghezza, in questo modo scorriamo insieme tutti gli array contenenti i dati dei vari luoghi